### PR TITLE
V1/62 create poll

### LINKED ISSUE #62: Create poll
As the presenter,
I want to create a poll
So my attendees can answer it.

- [x] Vuetify

- [x] Create the script part of the Vue

- [x] Send data on the form to the backend

- [x] request for adding the  choices

- [x] Test

- [x] Check the guidelines

**Estimation: 3h**

### PR DESCRIPTION
**Pull request by Lindwing**
# How to test
Have an authenticated user with existing presentation owned by him.
The route is `/sondage/:idPresentation/creer`

### CODE REVIEW

**Review by SundayBraid — COMMENTED** [review]
## Test

with the GUI I have test the following

- [x] User can create a poll with choice
- [x] Submit button is disable if a field is empty
- [x] HTTP post request are correctly send **Back end feature aren't merge into this branch**
- [x] Messages are showed to notify of success or failure while the execution

## Questions

What if the poll has more or less than 4 answers (eg. true or false) ?

**Comment by Lindwing** [comment]
It is not practical to lock with 4 choices. I have changed the component. We can create polls with 2 choices or more. I fixed the maximum to 6.

**Review by SundayBraid — COMMENTED** [review]
## Test

- [ ] I cannot create empty presentation
**i can submit a empty poll if I don't fill any box**
image

- [x] The form send correct number of post request regarding of the number of choice are display (according that all choice form must be filled)

**Comment by Lindwing** [comment]
Changed in c2df2d0

File: src/resources/js/components/CreatePoll.vue [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      label="choix 5"
+      required
+    />
+
+    <v-text-field v-if="nbChoice > 5"
+      v-model="inputChoice.choice6"
+      :rules="choiceRules"
+      label="choix 6"
+      required
+    />
```
Comment (fabio10000) [inline src/resources/js/components/CreatePoll.vue]: the number of choices should not be hardcoded

File: src/resources/js/components/CreatePoll.vue [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+      },
+      inputChoice: {
+        choice1: '',
+        choice2: '',
+        choice3: '',
+        choice4: '',
+        choice5: '',
+        choice6: '',
+      },
+      nbChoice: '',
```
Comment (fabio10000) [inline src/resources/js/components/CreatePoll.vue]: same here should not be hardcoded

**Review by fabio10000 — CHANGES_REQUESTED** [review]
maybe you should drop the issue of creating choices and leave it to someone else and just make a creation of poll

**Comment by Lindwing** [comment]
I remove the create choice part of the component, I don't know-how make this without hardcoded.

**Review by fabio10000 — APPROVED** [review]

**Review by SundayBraid — APPROVED** [review]
I have re-test the create poll feature with recent change

- [x] User can create a poll
- [x] Submit button is disable if the field is empty
- [x] HTTP post request are correctly send
- [x] Messages are showed to notify of success or failure while the execution

**Pull request merged**